airflow/dev/docker-compose-data-ingestion.yml

```diff
@@ -29,6 +29,21 @@ services:
             # Ensure that the executing shell sets a GCP_KEY_PATH value. (export GCP_KEY_PATH=/absolute/path/to/service/account/key.json)
             - ${GCP_KEY_PATH}:/root/keys/keyfile.json:ro
 
+    exporter:
+        image: het/exporter:latest
+        depends_on:
+            - scheduler
+            - webserver
+        environment:
+            - PORT=8080
+            - PROJECT_ID=${PROJECT_ID}
+            # Ensure that the executing shell sets an EXPORT_BUCKET value. (export EXPORT_BUCKET=my-test-bucket)
+            - EXPORT_BUCKET=${EXPORT_BUCKET}
```

Comment:
Can you add the EXPORT_BUCKET env variable to the README? I just added a section today for env variables that should be set when running locally. You'll see them listed under the "Set up" sub-header.

Reply:
Done! Thanks for catching!